Rovers/Rovers.tex

```diff
@@ -0,0 +1,249 @@
+It is very easy to over estimate the team's ability especially when the project is so new to the team.
+
+\subsection{Traceability}
+\label{subsec:traceability}
+
+It is important that the team members document the sources they use to gather information. 
+There will be a Google Drive folder that will hold notes with links to any books, articles, media or their sources that are relevant to the rover project. 
+The software and hardware will be tracked on GitHub. 
+This includes the technical report.
+
+\subsection{Assessing Technical Skills}
+\label{subsec:assessing-technical-skills}
+
+Building a rover, especially one built to URC specifications requires a large team with a diverse set of technical skills. 
+This project will require SPEX to look at and asses what areas are sufficient and what areas need to be developed in order to build a rover. 
+It will also let us know to what spec we are currently capable of building such a rover. 
+
+\subsection{Accessibility}
+\label{subsec:plug-n-play}
+
+The project would be very ambitious. It would require at a minimum a handful (4-5) members each with different areas of expertise. 
+The members would need that knowledge as well as \LaTeX{}, notetaking, and good research practices. 
+This would be to deliver the MVP.
+A larger team would be able to accomplish more.  
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project would involve divvying out the different project areas to the different team members or small groups and each week meeting to discuss progress. 
+The team would take this information, refine it and put it into the technical report along with construction.  
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The primary deliverables of this project will be the technical report and the rover itself by the end of the second semester. 
+It is also noted that the materials that the team will come across or notes should also be saved in an Google Drive folder for future reference.  
+The team will also present at SPEX design reviews and the weekly checkups at general meetings with the areas the team members are working on that week.
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+The largest milestones for the technical report would be the would be draft and completion. 
+The rover itself would include the various CAD models of the rover, software milestones, electrical system completion, and drive train system. 
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
```

Comment:
You covered parts of this section previously.

Another question to consider is 
>What is the minimum number of people with specific skills needed for this project to be successful

as well as

> What is the maximum number of people this project can reasonably support.

and

>What features/developments would be expanded upon if more than the minimum and less than the maximum people join the project?

What are the critical skills needed that if not had will mean the project can't proceed?